src/framework/preferences/js/Enactors.js

```diff
@@ -427,4 +427,244 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
+                    noMatchFlag = false;
+                    break;
+                }
+            }
+            // If it is not already in the records its color and background-color are added as arrays
+            if (noMatchFlag) {
+                var color = $(element).css("color");  // get the color as "rgb(..., ..., ...)"
+                color = color.slice(4, -1);
+                color = color.split(", ");  // color becomes and array with the 3 color components as his elements
+                var backgroundColor = $(element).css("background-color"); // the color can be is rgb or rgba format
+                if (backgroundColor.substr(0,4) === "rgba") {
+                    backgroundColor = backgroundColor.slice(5,-1);
+                }
+                else {
+                    backgroundColor = backgroundColor.slice(4,-1);
+                }
+                backgroundColor = backgroundColor.split(", ");
+                initialColorsDictionary.push({
+                    key: this,
+                    color: color,
+                    backgroundColor: backgroundColor
+                });
+            }
+            noMatchFlag = true;
+        });
+
+        var initialColors = $.extend(true, [], initialColorsDictionary);  // make a deep copy of the original array in order to prevent changes in the initial array
+        elements.each(function () {
+            // Finds the element in the dictionary
+            for (var count in initialColors) {
+                if (initialColors[count].key == this) {
+                    index = count;
+                    break;
+                }
+            }
+            var color = initialColors[index].color;  //get the color of the current component
+            color[2] = Math.round(color[2] * (1 - times)) + "";  //blue component
+            var colorToSet = "rgb(" + color[0] + ", " + color[1] + ", " + color[2] + ")";  //make the color in a "rbg(..,..,..)" format
+            $(this).css("color", colorToSet);
+            var backgroundColor = initialColors[index].backgroundColor;
+            backgroundColor[2] = Math.round(backgroundColor[2] * (1 - times)) + "";
+            if (backgroundColor[3] === undefined) {
+                var backgroundColorToSet = "rgb(" + backgroundColor[0] + ", " + backgroundColor[1] + ", " + backgroundColor[2] + ")";
+            }
+            else {
+                var backgroundColorToSet = "rgba(" + backgroundColor[0] + ", " + backgroundColor[1] + ", " + backgroundColor[2] + ", " + backgroundColor[3] + ")";
+            }
+            $(this).css("background-color", backgroundColorToSet);
+        });
+    };
+
+    fluid.prefs.enactor.blueColorFilter.addMutationObserver = function (applyFilter) {
+        // Uses document ready, because the script is added before the body
+        $(document).ready(function (){
+            var isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
+            if (isSafari) {
```

Comment:
This if statement had to fix an issue with the mutationObserver, because older versions of Safari required a webkit prefix, but it did not fix the issue. I will delete the Safari check and leave the issue open.